lib/appsignal/utils.rb

```diff
@@ -52,5 +52,15 @@ def encode_utf8(value)
 
       extend ClassMethods
     end
+
+    class Gzip
+      module ClassMethods
+        def compress(body)
+          Zlib::Deflate.deflate(body, Zlib::BEST_SPEED)
+        end
+      end
```

Comment:
If we drop the `ClassMethods` module here, we can do the same thing without having to `extend`:

``` ruby
class Gzip
  def self.compress(body)
    Zlib::Deflate.deflate(body, Zlib::BEST_SPEED)
  end
end
```